src/arch/cuda/hybridAPI/hapi_functions.h

```diff
@@ -38,13 +38,11 @@ AMPI_CUSTOM_FUNC(cudaError_t, hapiMalloc, void**, size_t)
 AMPI_CUSTOM_FUNC(cudaError_t, hapiFree, void*)
 AMPI_CUSTOM_FUNC(cudaError_t, hapiMallocHost, void**, size_t)
 AMPI_CUSTOM_FUNC(cudaError_t, hapiFreeHost, void*)
-AMPI_CUSTOM_FUNC(cudaError_t, hapiMallocHostPool, void**, size_t)
-AMPI_CUSTOM_FUNC(cudaError_t, hapiFreeHostPool, void*)
```

Comment:
These wrappers to `hapiPool*` functions are no longer needed as `hapiPool*` functions also return `cudaError_t` and have the same function prototypes as the CUDA variants.